.github/CODEOWNERS

```
@@ -0,0 +1,31 @@
+# Default reviewers
+* @allyhawkins @jashapiro @jaclyn-taroni @sjspielman
```

I think we should start with one person getting assigned who will be tasked with triage:

```suggestion
* @allyhawkins
```